adafruit_logging.py

```diff
@@ -41,54 +71,56 @@
     (50, "CRITICAL"),
 ]
 
-for value, name in LEVELS:
-    globals()[name] = value
+for __value, __name in LEVELS:
+    globals()[__name] = __value
 
 
-def level_for(value):
-    """Convert a numberic level to the most appropriate name.
+def level_for(level_value):
+    """Convert a numeric level to the most appropriate name.
 
-    :param value: a numeric level
+    :param int level_value: a numeric level
 
     """
     for i in range(len(LEVELS)):
-        if value == LEVELS[i][0]:
+        if level_value == LEVELS[i][0]:
```

Comment:
why did you change `value` to `level_value`? Is it conflicting with another variable name?

Reply:
Ok, thanks for the explanation, resolving